client/app/shared/tab-page/tab-page.component.ts

```diff
@@ -16,20 +16,20 @@ export class TabPageComponent extends ContainerComponent {
     this.factoryRef = injector.get(InputFactoryService);
   }
 
-  AddChildElements(factoryRef, containerRef, itemArray) {
+  AddChildElements(factoryRef, containerRef, itemArray, viewNumber) {
     if (this.modelRef) {
       for (let item of itemArray) {
         let childModelRef = this.modelRef[item.relVal];
-        factoryRef.createComp(containerRef, item, childModelRef);
+        factoryRef.createComp(containerRef, item, childModelRef, viewNumber);
```

Comment:
viewNumber is added as a third param in createComp function in file: input-factory.service.ts